service/controller/resource/alertmanagerconfig/test/case-0-cluster-api.golden

```diff
@@ -0,0 +1,13 @@
+metadata:
+  creationTimestamp: null
+  name: alertmanager-config
+  namespace: bob-prometheus
+stringData:
+  alertmanager-additional.yaml: |
```

Comment:
Also, should the content not be encoded here?

Reply:
Yeah that's right